charts/cortx/templates/_images.tpl

```diff
@@ -67,7 +67,7 @@ Return the CORTX setup initContainer
   {{- if eq $image "ghcr.io/seagate/centos:7" }}
     - sleep $(shuf -i 5-10 -n 1)s
   {{- else }}
-    - /opt/seagate/cortx/provisioner/bin/cortx_deploy -f /etc/cortx/solution -c yaml:///etc/cortx/cluster.conf
+    - /opt/seagate/cortx/provisioner/bin/cortx_deploy -f /etc/cortx/solution -c $CONFSTORE_URL
```

Comment:
Did you want to expand out the long names for the config switches here (this one may have slipped through since you have it on the rest of them)?

Reply:
I sure do.

Reply:
Oh, just remembered, `cortx_deploy` does not support long parameters. 😢